src/models/devicemodel.h

```diff
@@ -22,7 +22,7 @@ class DeviceModel final : public QAbstractListModel {
   DeviceModel();
   ~DeviceModel();
 
-  enum ServerCountryRoles {
+  enum ModelRoles {
```

Comment:
Why did you rename this?

Reply:
because "ServerCountryRoles" is not a good name for a "DeviceModel".